src/test/java/org/broadinstitute/hellbender/tools/walkers/GnarlyGenotyperIntegrationTest.java

```diff
@@ -35,7 +35,7 @@
         return new Object[][]{
                 // Simple Test, spanning deletions; standard calling confidence
                 //No variants outside requested intervals; no SNPs with QUAL < 60, no INDELs with QUAL < 69?; has star alleles after deletion at chr20:263497; has AC, AF, AN, DP, ExcessHet, FS, MQ, (MQRankSum), (ReadPosRankSum), SOR, QD; has called genotypes
-                {new File[]{getTestFile("sample1.vcf"), getTestFile("sample2.vcf"), getTestFile("sample3.vcf"), getTestFile("sample4.vcf"), getTestFile("sample5.vcf")},
+ /*               {new File[]{getTestFile("sample1.vcf"), getTestFile("sample2.vcf"), getTestFile("sample3.vcf"), getTestFile("sample4.vcf"), getTestFile("sample5.vcf")},
```

Comment:
@ldgauthier was rebasing and updating test files for #7394 and noticed you removed these tests (which were previously failing in my branch due to the ExcessHet updates). Didn't see any review comments here providing further info, although I might've missed them elsewhere---was this removal intentional? If not, should I restore and update ExcessHet values + other diffs resulting from your branch? If so, should I just leave them be?

In any case, I'm going to add an `UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS` toggle for these tests.